app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
Using a multi-line import with parentheses is generally better than the single-line, backslash-continued style. It is more readable, avoid syntax issues, it's more consistent with PEP8 and easier to maintain